documentapi/src/main/java/com/yahoo/documentapi/SyncSession.java

```diff
@@ -45,10 +45,10 @@
      * @throws UnsupportedOperationException Thrown if this access does not
      *                                       support retrieving.
      */
-    Document get(DocumentId id);
+    default Document get(DocumentId id) { return get(id, null); }
```

Comment:
Will all existing implementations of `SyncSession.get(DocumentId,Duration)` accept `null` as timeout?

Reply:
Yes (and it would be a bug otherwise as the Javadoc specifies the behavior in that case)